data/entities.json5

```diff
@@ -11532,5 +11532,10 @@
     categories: ['marketing'],
     domains: ['*.trychameleon.com'],
     examples: ['fast.trychameleon.com'],
+  },
+  {
+    name: 'Matomo',
+    categories: ['analytics'],
+    domains: ['*.matomo.cloud', '*.analytics.ozzi.io'],
```

Comment:
What's this second domain? is it really the same? looks like a separate company and half the links are broken on [their homepage](https://www.ozzi.io/)

Reply:
.analytics.ozzi.io is a cloud provider like .matomo.cloud - the website ozzi.io website itself is quite broken though...

Reply:
I understand they are another provider but are they the *same business entity* as matomo.cloud? It doesn't appear to be. It looks like just another company that has taken the open source analytics and served it on another origin.

If that's true, we don't want them in the same entity here. For example, jQuery is served over more than a dozen different CDNs we list. We don't consolidate because they all serve the same files, we split them out to understand *who* is serving those files :)

Reply:
I was looking at how Piwik was identified - the domain there have nothing to do with Piwik - just providers that offer Piwik.

Reply:
But we can remove ozzio.io for now, and I will update Piwik in another pull request :)

Reply:
alright then :)

```suggestion
    domains: ['*.matomo.cloud'],
```